multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/security/SpaceNameBasedAuthorizationFilter.java

```diff
@@ -21,21 +22,33 @@ public abstract class SpaceNameBasedAuthorizationFilter implements UriAuthorizat
 
     private final AuthorizationChecker authorizationChecker;
 
+    @Inject
+    private LoginAttemptAuditLog loginAttemptAuditLog;
```

Comment:
Make `loginAttemptAuditLog` field protected inside abstract class and just reuse it. Convert into constructor injection.